pulp_rpm/app/viewsets.py

```diff
@@ -115,15 +116,170 @@ class PackageViewSet(SingleArtifactContentUploadViewSet):
     minimal_serializer_class = MinimalPackageSerializer
     filterset_class = PackageFilter
 
+    DEFAULT_ACCESS_POLICY = {
+        "statements": [
+            {
+                "action": ["list", "retrieve"],
+                "principal": "authenticated",
+                "effect": "allow",
+            },
+            {
+                "action": ["create"],
+                "principal": "authenticated",
+                "effect": "allow",
+            },
+        ],
+    }
+
 
-class RpmRepositoryViewSet(RepositoryViewSet, ModifyRepositoryActionMixin):
+class RpmRepositoryViewSet(RepositoryViewSet, ModifyRepositoryActionMixin, RolesMixin):
```

Comment:
Do you think this file would be more or less readable if it was broken up into multiple files?  Currently `viewsets.py` has all the viewsets in one file, whereas that's not true of models or serializers.

I could think of two different ways of doing it, putting all the access policies in one file with names like `RPM_REPOSITORY_DEFAULT_ACCESS_POLICY`, or splitting up the viewsets like we do with models and serializers.

Or, if you think it wouldn't help, just leave it as-is.

Reply:
I like the idea of splitting it as the models are. But I believe it is for another PR.

Reply:
Also, for now, I think it will be better to keep policies with viewsets together. To have them visible in one place.